src/util/Util.js

```diff
@@ -101,6 +108,34 @@ define(['okta', 'util/Logger'], function (Okta, Logger) {
     Logger.warn(`\n${message.replace(/^(\s)+/gm, '')}`);
   };
 
+  // Heleper function to submit a url via POST request.
+  // what it does is actually create a hidden form
+  // and fill values from the url (base url, query parameters)
+  // and submit the form.
+  Util.postToUrl = function (url, $el) {
```

Comment:
Can we rename this to be more specific: something like `createPostFormAndSubmit` ?

Reply:
hmm, why would you prefer more specific naming that indicates a little more implementation detail?
I thought `postToUrl` explain what it does?

Reply:
Maybe `redirectWithForm` or `redirectToUrlWithPost`? I like the simplicity of `postToUrl`, but to Santhosh's point I think `postToUrl` is probably to general.... or maybe keep `postToUrl` but not have this method in a Util file since it is very specific for IdpDiscovery?

Reply:
^ exactly my thoughts.

Reply:
vote for `redirectToUrlWithPost`